tools/gitlab.md

```diff
@@ -0,0 +1,64 @@
+---
+permalink: /gitlab
+layout: post
+title: GitLab
+command: gitlab-rake gitlab:env:info
+link: https://docs.gitlab.com/ce/policy/maintenance.html
+changelogTemplate: https://gitlab.com/gitlab-org/gitlab/-/releases/v__RELEASE_CYCLE__.0-ee
+releaseDateColumn: true
+sortReleasesBy: "releaseCycle"
+activeSupportColumn: true
+eolColumn: Maintenance Support
+iconSlug: gitlab
+# The release date for "N" should match the eol date for N-3 release.
+releases:
+  - releaseCycle: "14.1"
+    release: 2021-07-22
+    eol: 2021-10-22
+    latest: "14.1.1"
+    support: 2021-08-22
+  - releaseCycle: "14.0"
+    release: 2021-06-22
+    latest: "14.0.6"
+    eol: 2021-09-22
+    support: 2021-07-22
+  - releaseCycle: "13.12"
+    release: 2021-05-22
+    latest: "13.12.8"
+    eol: 2021-08-22
+    support: 2021-06-22
+  - releaseCycle: "13.11"
+    release: 2021-04-22
+    latest: "13.11.7"
+    support: 2021-05-22
+    eol: 2021-07-22
+  - releaseCycle: "13.10"
+    release: 2021-03-22
+    latest: "13.10.5"
+    support: 2021-04-22
+    eol: 2021-06-22
+
+---
+> [GitLab](https://about.gitlab.com/) is a web-based DevOps lifecycle tool that provides a Git repository manager providing wiki, issue-tracking and continuous integration and deployment pipeline features, using an open-source license, developed by GitLab Inc.
+
+GitLab has strict policies governing version naming, as well as release pace for major, minor, patch, and security releases. New releases are announced on the [GitLab blog](https://about.gitlab.com/releases/categories/releases/).
```

Comment:
Can this paragraph be dropped? Doesn't add much information, except for the link, which can be better put at the bottom (Is there a mailing list or RSS feed for it? That would work better).

```suggestion
```